lib/Backends/CPU/Transforms.cpp

```diff
@@ -34,11 +34,12 @@ using llvm::isa;
 static Node *optimizeCPUConv(ConvolutionNode *CN, Function *F) {
   auto depth = CN->getFilter().dims()[0];
   auto *M = F->getParent();
+  auto group = CN->getGroup();
 
-  // The depth dimension must be a multiple of 64 to perform the
+  // The depth dimension in each group must be a multiple of 64 to perform the
   // transformation. This transformation is currently only profitable on
   // low-channel convolutions.
-  if (depth < 64 || (depth % 64) != 0) {
+  if (depth < 64 || ((depth / group) % 64) != 0) {
```

Comment:
Why not `(depth / group) < 64`?

Reply:
@artemrakhov The code processes the weight in chunks of 64 elements.

Reply:
I understand. 

We know that depth is greater than 0 and is divisible by group.
Condition `((depth / group) % 64) != 0` (or `(depth % 64) != 0`) includes `(depth / group) < 64` (or `group < 64`). Because the smallest number divisible by 64 is 64.

In other words, my point is that the whole line could if replaced with:
```
  // Make sure that the depth group is divisible by 64.
  bool divisibleBy64 = ((depth / group) % 64) == 0;
  if (!divisibleBy64) {
```

Reply:
Oh yes!  In that case, we can just write:

  // Make sure that the depth group is divisible by 64.
  if (((depth / group) % 64) != 0) {